src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -494,7 +494,7 @@ protected override string GetDescription(IReadOnlyList<string> nameParts)
                 {
                     // It does not matter if it is fully qualified or simple so lets return the simple name.
                     return root.AddUsingDirective(
-                        simplifiedUsing.WithoutTrivia().WithoutAnnotations(), contextNode, placeSystemNamespaceFirst,
+                        simplifiedUsing.WithoutAnnotations(), contextNode, placeSystemNamespaceFirst,
```

Comment:
Ah, looking at the while file, I see that this is inside of `if (UsingsAreContainedInNamespace(contextNode))`.

---
In reply to: [58461151](https://github.com/dotnet/roslyn/pull/10188#discussion_r58461151) [](ancestors = 58461151)